cmp/dataview/DataViewModel.js

```diff
@@ -125,13 +125,19 @@ export class DataViewModel extends HoistModel {
     get groupBy()               {return this.gridModel.groupBy}
     get sortBy()                {return this.gridModel.sortBy}
```

Comment:
^ *Very* minor OCD thing, but we should probably tab these over so they align with the methods below